CHANGELOG.md

```diff
@@ -3,6 +3,56 @@
 All notable changes to this project will be documented in this file.
 
 
+## Unreleased - as of Sprint 37 end 2016-03-07
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+37+Ending+Mar+7%2C+2016%22+label%3Aenhancement)
+
+- Providers
+  - Vmware: Clustered datastores
+  - Containers: Bug fixes and UI updates
+  - Microsoft SCVMM: Inventory Performance
+  - Red Hat OpenStack: Instance Evacuation, better neutron modeling, Ceilometer events
+  - Pluggable providers: Amazon extraction
+- Tenancy
+  - Added scoping strategy for provision request
+  - Added ability to report on tenants and tenant quotas
+- REST API
+  - Added support for Service Reconfigure action
+  - Added new service orders collection and CRUD operations
+- SSUI
+  - RBAC Control of Menus and Features
+  - Reconfiguring a Service
+  - Set Ownership of a Service
+ - Automate
+  - Ansible Service Models
+  - Added Google Auto-Placement methods
+
+### [Changed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+37+Ending+Mar+7%2C+2016%22+label%3Aenhancement)
+
+- Core
+  - Updated to Rails 5 (using Rails master branch until stable branch is cut)
+  - Replacement of rubyrep with pglogical in progress
+- Appliance Console
+  - Configure Database
+  - Change Region
```

Comment:
Exposed options in the appliance console for configuring and resetting a database (essentially [this](https://github.com/ManageIQ/manageiq/blob/master/gems/pending/appliance_console/locales/en.yml#L13-L14))